### DISCUSSION TITLE
Seaweed Master question

### DISCUSSION THREAD

**Discussion by lonnietc** [seaweedfs/seaweedfs#2410] [Q&A]
Hello,

My testing with Seaweed is going very well and I just have a few more things to figure out.

With regards to the Seaweed Master Server.

On the github site it says to start a master:

./weed master

Now then, when that run, I am assuming that it is a type of controller for the volume nodes.

Question:  Can, or should, you run multiple Master nodes (on different systems) in case of a master failure?

Thanks,

**Comment by chrislusf** [comment]
(Moving this issue to discussions.)

Yes. https://github.com/chrislusf/seaweedfs/wiki/Production-Setup#setup-masters
